### PR TITLE
(feat:plugin-user): add prelogin ID to user activate request as header

### PR DESCRIPTION
**Pull request by Cruig**

### CODE REVIEW

File: packages/node_modules/@ciscospark/plugin-user/src/user.js [hunk 1]
```diff
@@ -261,6 +262,11 @@ const User = SparkPlugin.extend({
     let requiresClientCredentials = false;
     let headers;
 
+    if (options.preloginId) {
+      headers = {'X-Prelogin-UserId': options.preloginId};
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/plugin-user/src/user.js]: please use lowercase for header keys

Comment (Cruig) [inline packages/node_modules/@ciscospark/plugin-user/src/user.js]: fixed

**Comment by ianwremmel** [comment]
Can this be merged or closed?

**Comment by Cruig** [comment]
@ianwremmel just pushed to Gauntlet

**Pull request closed without merging**